src/Resources/config/services.yaml

```diff
@@ -9,10 +9,23 @@ services:
         class: Assoconnect\MJMLBundle\Compiler\Compiler
         arguments:
             $customCompiler: "@assoconnect_mjml.custom_compiler"
-            $mjmlCompiler: "@assoconnect_mjml.mjml_compiler"
+            $mjmlCompiler: "@assoconnect_mjml.mjml_compiler_service"
             $cacheDir: "%kernel.cache_dir%"
-            $projectDir: "%kernel.project_dir%"
+            #$projectDir: "%kernel.project_dir%"
+
+    assoconnect_mjml.rest_mjml_compiler_client:
+        class: GuzzleHttp\Client
+        public: true
+        arguments:
+            - base_uri: '%assoconnect_mjml.rest_mjml_compiler_host%'
 
+    assoconnect_mjml.mjml_compiler_service:
```

Comment:
why create an alias?

Reply:
taht is part of the external rest compiler, will be part of the other PR